### PR TITLE
added CKAN 2.9 commands for harvesterext to README

### PR DESCRIPTION
**Pull request by andrecastro0o**
Concerning https://github.com/ckan/ckanext-harvest/issues/423

I have tested all the commands. 
Nothing seems to have change in the commands but the way they are invoked.

Hope this is helpful. Let me know if something needs adjusting.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -92,7 +92,13 @@ config option (or ``default``) will be used to namespace the relevant things:
 Configuration
 =============
 
-Run the following command to create the necessary tables in the database (ensuring the pyenv is activated)::
+Run the following command to create the necessary tables in the database (ensuring the pyenv is activated):
+
+ON CKAN == 2.9::
```
Comment (amercader) [inline README.rst:97]: Let's future-proof this!

```suggestion
ON CKAN >= 2.9::
```

Comment (amercader) [reply README.rst:97]: Same on all other `CKAN == 2.9` occurrences

**Review by amercader — COMMENTED** [review]
@andrecastro0o This looks great, only one minor correction please

File: README.rst (hunk 1 above)
Comment (andrecastro0o) [reply README.rst]: Thanks @amercader . Makes sense. Please see 6ab5ca4

**Comment by amercader** [comment]
Thanks @andrecastro0o

**Pull request merged**